airflow/models.py

```diff
@@ -3399,7 +3399,8 @@ def run(
             donot_pickle=configuration.getboolean('core', 'donot_pickle'),
             ignore_task_deps=False,
             ignore_first_depends_on_past=False,
-            pool=None):
+            pool=None,
+            conf=None):
```

Comment:
Personally prefer breaking the line on the last arg and leaving a trailing coma. This allow you to only be on the `git blame` for the line your touch

Reply:
Good to know!